### PR TITLE
Feat/media query 3.1

### PR DESCRIPTION
**Pull request by ala-n**

### CODE REVIEW

**Review by julia-murashko — APPROVED** [review]

**Review by NastaLeo — APPROVED** [review]

**Review by andreybelous — APPROVED** [review]

**Review by nattallius — APPROVED** [review]
Could you also correct jsdocs for `parse`, `parseTuple`?

**Review by dshovchko — APPROVED** [review]

**Review by NastaLeo — APPROVED** [review]

**Review by abarmina — APPROVED** [review]

File: src/modules/esl-media-query/core.ts [hunk 1]
```diff
@@ -3,4 +3,7 @@ export * from './core/common/screen-dpr';
 export * from './core/common/screen-breakpoint';
 
 export * from './core/esl-media-query';
+export * from './core/esl-media-rule';
 export * from './core/esl-media-rule-list';
+
+// export * from './core/decorator/media';
```
[the remark this answers is not in the document]
Comment (ala-n) [reply src/modules/esl-media-query/core.ts:9]: Yep, temporary, while API is under discussion

**Pull request merged**